bigquery/query_places.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Places
+nav_order: 1
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Places
+
+## Place containment
+
+* List all the places of type X contained in Y, e.g. counties in USA:
```

Comment:
Nit: suggest using ### or #### so we can generate a TOC for these pages.

Reply:
Done